suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -88,35 +88,39 @@ export const XpubScanScreen = ({
     };
 
     return (
-        <Screen header={<AccountImportHeader activeStep={2} />}>
-            <Card>
-                <SelectableNetworkItem
-                    cryptoCurrencyName={networks[selectedCurrencySymbol].name}
-                    cryptoCurrencySymbol={selectedCurrencySymbol}
-                    iconName={selectedCurrencySymbol}
-                    onPressActionButton={() => navigation.goBack()}
-                />
-            </Card>
-            <View style={applyStyle(cameraStyle)}>
-                <XpubImportSection onRequestCamera={handleRequestCamera} />
-            </View>
-            <TextDivider title="OR" />
-            <Form form={form}>
-                <VStack spacing="medium">
-                    <TextInputField name="xpubAddress" label="Enter x-pub..." />
-                    <XpubHint networkType={networks[selectedCurrencySymbol].networkType} />
-                    <Button
-                        onPress={onXpubFormSubmit}
-                        size="large"
-                        isDisabled={!watchXpubAddress?.length}
-                    >
-                        Submit
-                    </Button>
-                </VStack>
-            </Form>
-            {isDevelopOrDebugEnv() && (
-                <DevXpub symbol={selectedCurrencySymbol} onSelect={goToAccountImportScreen} />
-            )}
+        <Screen
+            header={<AccountImportHeader activeStep={2} />}
+            footer={<XpubHint networkType={networks[selectedCurrencySymbol].networkType} />}
+        >
+            <ScrollView>
```

Comment:
ScrollView is already inside of Screen component, why we need another one here?

Reply:
removed in https://github.com/trezor/trezor-suite/commit/a2d3117fe98d90bb542c026df2b20fa1beb7c583